src/main/java/core/gui/theme/ImageUtilities.java

```diff
@@ -581,6 +581,19 @@ public static Icon getSmileyIcon(String smileyName) {
 		return null;
 	}
 
+	public static Icon getSmallPlayerSpecialtyIcon(String playerSpecialtyName) {
+		return getPlayerSpecialtyIcon(playerSpecialtyName, 15);
+	}
+
+	public static Icon getPlayerSpecialtyIcon(String playerSpecialtyName, int size) {
+		if (Arrays.stream(HOIconName.SPECIALTIES).skip(1).anyMatch(playerSpecialtyName::equals)) {
```

Comment:
I believe `ThemeManager` was caching those images, we probably should make sure those ones are cached as well, as it has caused display perf issues in the past.

Reply:
I have another issue at the moment with this one that I can't explain in Lineup tab.

**Before:**

![image](https://user-images.githubusercontent.com/1136496/97918709-d4d42d00-1d56-11eb-9980-73a7b60b71c9.png)


With this commie:
![image](https://user-images.githubusercontent.com/1136496/97918784-f503ec00-1d56-11eb-89b3-21fabae6f73b.png)


The bottom line is truncated. I can't figure out what I did that is causing it. Looks like the preferred size is now wrongly calculated.

Reply:
if you have an idea of what is going on ....

Reply:
I have the same problem after switching to darklaf 2.5.x -- reverting back to 2.4.x fixes the issue.

Reply:
>I have the same problem after switching to darklaf 2.5.x -- reverting back to 2.4.x fixes the issue

indeed you are right ...  @weisJ any idea what is going on?

Reply:
> I believe ThemeManager was caching those images, we probably should make sure those ones are cached as well, as it has caused display perf issues in the past.

good catch ... I added the caching